src/stream/src/common/log_store_impl/kv_log_store/reader.rs

```diff
@@ -672,6 +598,107 @@ impl<S: StateStoreRead + Clone> LogReader for KvLogStoreReader<S> {
     }
 }
 
+#[allow(clippy::too_many_arguments)]
```

Comment:
Are the `read_flushed_chunk` and `read_persisted_log_store` just a simple copy-paste from the original code without any change in the logic?

Reply:
Yes. These are refactored and reused in the synced_log_store executor.